src/Scene/LayersConfiguration.js

```diff
@@ -18,6 +18,12 @@ function LayersConfiguration() {
     this.geometryLayers = [];
     this.colorLayers = [];
     this.elevationLayers = [];
+    this.lightingLayers = [];
```

Comment:
I still think it's broken to add a layer simply to be able to initialize a uniform value.
Note that this is the same problem @autra has in its PR debug ("how to initialize the showOutline value?")...

Anyway, I believe #239 will allow to make something simpler, such as allowing the user to define a function called each time a new time a new tile is created.

So I propose that we merge both PR as is, and revisit the 2 issues in the refactoring PR.
Makes sense?